ethcore/src/client/client.rs

```diff
@@ -1245,7 +1245,9 @@ impl BlockChainClient for Client {
 	}
 
 	fn chain_info(&self) -> BlockChainInfo {
-		self.chain.read().chain_info()
+		let mut chain_info = self.chain.read().chain_info();
+		chain_info.pending_total_difficulty = chain_info.total_difficulty + self.block_queue.total_difficulty();
```

Comment:
can't we often have multiple competing blocks from different forks in the queue? (i.e. this value will often be higher than the actual total difficulty of the best queued block)